superset/utils/decorators.py

```diff
@@ -32,6 +32,27 @@
     from superset.stats_logger import BaseStatsLogger
 
 
+def statsd_gauge(metric_prefix: Optional[str] = None) -> Callable[..., Any]:
+    def decorate(f: Callable[..., Any]) -> Callable[..., Any]:
+        """
+        Handle sending statsd gauge metric from any method or function
+        """
+
+        def wrapped(*args: Any, **kwargs: Any) -> Any:
+            metric_prefix_ = metric_prefix or f.__name__
+            try:
+                result = f(*args, **kwargs)
+                current_app.config["STATS_LOGGER"].gauge(f"{metric_prefix_}.ok", 1)
```

Comment:
Does this allow for the end user to set a custom prefix?

Reply:
yes, when creating a custom statsd logger class